src/server/pps/server/api_server.go

```diff
@@ -3111,3 +3116,21 @@ func labels(app string) map[string]string {
 		"component": "worker",
 	}
 }
+
+func (a *apiServer) authCheckPachdLogs(ctx context.Context) error {
```

Comment:
This would probably make sense as a non-transaction verson of `CheckClusterIsAuthorizedInTransaction`, similar to `CheckRepoIsAuthorized` (https://github.com/pachyderm/pachyderm/blob/master/src/server/auth/server/util.go#L12). But we could do that in a follow-up, up to you.

Reply:
good point. I'll update it here